clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/MigrateSecurityGroupStrategy.java

```diff
@@ -0,0 +1,177 @@
+/*
```

Comment:
@cfieber My attempt at using an interface (and also using Java 8)...

I am pretty sure I missed the point of using an interface instead of an abstract class.